content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,98 @@
+---
+linkTitle: Project Export iOS
+title: Project Export for iOS
+description: (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS.
+toc: true
+weight: 450
+---
+This guide covers how to use project export tooling to build and deploy games on iOS. First, we will use the iOS export script to generate an Xcode project and prepare assets. Afterwards we will walk through how to ensure Xcode is properly setup to build an iOS app.
+
+For this demonstration, we will use the O3DE Atom Sample Viewer project as our reference point for ensuring proper behavior. Results may vary for other projects.
+
+{{< note >}}
+To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](project-export-cli)
+{{< /note >}}
+{{< important >}}
+The iOS export functionality is only available on macOS, which is currently experimental for O3DE.
+{{< /important >}}
+
+## Prerequisites
+1. Complete the [Project Export CLI Tool page.](project-export-cli)  prerequisites. 
+2. Git clone [O3DE Atom Sample Viewer project](https://github.com/o3de/o3de-atom-sampleviewer) onto your machine.
+3. Register [O3DE Atom Sample Viewer project](https://github.com/o3de/o3de-atom-sampleviewer) with O3DE. You can register the project using:
+```
+<O3DE_ENGINE>/scripts/o3de.sh register -pp /path/to/o3de-atom-sampleviewer
+```
+4. Have a valid copy of Xcode installed on your machine, and a valid Apple Developer ID associated with that IDE. To set up such an ID, go to [developer.apple.com](https://developer.apple.com). To use your desired Apple ID, in Xcode go to `Xcode -> Settings`. In the Settings window, go to the Accounts tab and click the '+' icon at the bottom of the left panel to add your account. Select Apple ID to proceed, and follow on-screen instructions.
+5. You will also need to ensure that Xcode has the standard SDKs installed for macOS and iOS, and that your preferred iOS test device is compatible with Xcode. If you are not sure, you can consult [this page on installing Simulator Runtimes](https://developer.apple.com/documentation/xcode/installing-additional-simulator-runtimes).
+
+## Quickstart
+### Running the Export Script
+Assuming you are just following along using the O3DE Atom Sampleviewer project, this single invocation should be all you need to generate the necessary Xcode project file in the project's build folder:
+```
+<O3DE_ENGINE>/scripts/o3de.sh export-project -es <O3DE_ENGINE>/scripts/o3de/ExportScripts/export_source_ios_xcode.py -pp /path/to/o3de-atom-sampleviewer
+```
+Assuming the underlying CMake build system finds no issues with your engine or project installation, you should have the corresponding Xcode project file, which will be located at `<PROJECT>/build/game_ios`. For the Atom SampleViewer, it should be called `AtomSampleViewer.xcodeproj`.
```

Comment:
is `/build/game_ios` the actual path you ended up with or did you change it to `/build/ios_mono` ?

Reply:
yes, `/build/game_ios` is the actual path I got. This is because it's the default folder name for ios output in the export script itself.